aspnetcore/blazor/debug.md

```diff
@@ -92,14 +92,6 @@ While debugging your Blazor WebAssembly app, you can also debug your server code
 
 ## Visual Studio Code
 
-To debug a Blazor WebAssembly app in Visual Studio Code:
- 
-Install the [C# extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp) and the [JavaScript Debugger (Nightly)](https://marketplace.visualstudio.com/items?itemName=ms-vscode.js-debug-nightly) extension with `debug.javascript.usePreview` set to `true`.
```

Comment:
I think we should keep the guidance on installing C# for new first-time Blazor users.

Reply:
We have that in the new Tooling doc. I recommend that we cross-link to it.

Reply:
It might change one day, and it would be easier to manage it that way.